src/server.rs

```diff
@@ -1582,6 +1596,33 @@ impl<R, B> Message<R, B> {
     }
 }
 
+struct BincodeCodec;
```

Comment:
github doesn't allow to comment on unpatches sections but there's a compile failure in `impl<R> futures_03::Stream for Body<R>` because the `receiver` changed type and doesn't have `poll` anymore. It does have `poll_next`, though, which simplifies the implementation.